content/src/content/jcr_root/apps/core/wcm/components/page/v3/page/dataLayer.html

```diff
@@ -0,0 +1,60 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+  
+  <sly data-sly-test="${currentPage.listChildren}">
+   <sly data-sly-list.childPage="${currentPage.listChildren @ begin = 0, end = 1}">
+        <sly data-sly-set.childPageTitle="${childPage.title}"/>
+       <sly data-sly-test="${childPage.listChildren}">
+           <sly data-sly-list.subchildPage="${childPage.listChildren @ begin = 0, end = 1}">
+               <sly data-sly-set.subchildPageTitle="${subchildPage.title}"/>
+               <sly data-sly-test="${subchildPage.listChildren}">
+                   <sly data-sly-list.tertiarychild="${subchildPage.listChildren @ begin = 0, end = 1}">
+                       <sly data-sly-set.tertiarychildTitle="${tertiarychild.title}"/>
+                   </sly>
+               </sly>
+           </sly>
+       </sly>
+    </sly>
+</sly>
+<!--/* Data layer JavaScript object: */-->
+<script data-sly-use.page="com.adobe.cq.wcm.core.components.models.Page">
+       window.digitalData = window.digitalData || {};
+       window.digitalData.page = window.digitalData.page || {};
+        window.digitalData.page.attributes = window.digitalData.page.attributes || { };

```

Comment:
Indentation/spacing is off throughout the PR.

Reply:
fixed